livre/Eq/Config.cpp

```diff
@@ -40,6 +40,12 @@
 #  include <zeq/hbp/hbp.h>
 #endif
 
+#ifdef LIVRE_USE_RESTCONNECTOR
+#  include <restconnector/RestConnector.h>
+static const std::string PUBLISHER_SCHEMA_SUFFIX = "resp://";
+static const std::string SUBSCRIBER_SCHEMA_SUFFIX = "cmd://";
```

Comment:
The schema names don't tell me anything.

1) Why not one
2) Why not reuse the hbp publisher/subscriber?